server/lorax_server/models/model.py

```diff
@@ -228,6 +228,10 @@ def check_initialized(self):
                 f"found uninitialized parameters in model {self.__class__.__name__}: {uninitialized_parameters}"
             )
 
+    @property
+    def supports_cuda_graph_compilation(self) -> bool:
+        return True
```

Comment:
Do we need to set this to False for some models?

Reply:
At least with the limited models I tested, compile worked so we don't set this property to False for any model, but we may want to in the future. Not needed at the moment so happy to get rid of it.

Reply:
Check for lines like this in the code:

```
if compile:
            raise ValueError("`--compile` is not supported with CausalLM")
```

There are a number of them